test/integration/sanity/Sanity.e2e.test.js

```diff
@@ -0,0 +1,15 @@
+// <reference types="Cypress" />
+describe('Preview Sanity', () => {
+    let token;
+
+    beforeEach(() => {
+        token = Cypress.env('ACCESS_TOKEN');
+        cy.visit('/');
+    });
+
+    it('Should load a document preview', () => {
+        const fileId = Cypress.env('FILE_ID_DOC');
+        cy.showPreview(token, fileId);
+        cy.getByTestId('preview-container').contains('The Content Platform for Your Apps', { timeout: 10000 });
```

Comment:
Is the timeout needed to account for fetching the file/rep? You should be able to use a combination of `cy.route` and `cy.wait` to avoid an arbitrary timeout.

Reply:
I tried the load event approach, but couldn't find a way to have Cypress wait for additional assertions within the 'load' event handler. Instead I wait for the `.bp` container to have the `.bp-loaded` class before attempting to assert the content